modules/swagger-codegen/src/main/java/io/swagger/codegen/CliOption.java

```diff
@@ -1,12 +1,25 @@
 package io.swagger.codegen;
 
+import io.swagger.models.properties.StringProperty;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
 public class CliOption {
     private final String opt;
     private String description;
+    private String type;
+    private String defaultValue;
+    private Map<String, String> enumValues;
 
     public CliOption(String opt, String description) {
+        this(opt, description, StringProperty.TYPE);
+    }
+
+    public CliOption(String opt, String description, String type) {
         this.opt = opt;
         this.description = description;
+        this.type = type;
     }
 
     public String getOpt() {
```

Comment:
Please rename this using `@ApiModelProperty` jackson annotation to `name` instead of `opt` in the JSON response